### ISSUE TITLE
No visual separation between draft and TOC

### ISSUE THREAD

**Issue by Abhayakara** [rfc-format/draft-iab-rfc-css-bis#4]
Right now the eye has no anchor to separate the TOC on the right from the text on the left, and emphasize the text, so there's a tendency to get drawn into the TOC when trying to read the text.   Shading the background behind the text, or boxing the TOC, would really help.

**Comment by martinthomson** [comment]
I agree that it's a potential problem, but the gutter is pretty big.

**Comment by Abhayakara** [comment]
The gutter isn't emphasized in any way, so my eye at least glides right
over it.
